packages/event-bus-redis/src/types/index.ts

```diff
@@ -0,0 +1,40 @@
+import { Job, JobsOptions, QueueOptions, WorkerOptions } from "bullmq"
```

Comment:
self-review: @adrien2p + @carlos-r-l-rodrigues Can I get you to validate my structuring of types and utils in `@medusajs/types` and `@medusajs/utils`? :)